docs/partner/index.md

```diff
@@ -1,48 +1,68 @@
-The following information will be used by LumApps to register your partner organization in the LumApps Marketplace and to generate your LumApps test platform.
-You'll receive all needed information to start working on your extension as soon as your environment is set up on LumApps.
+We may reject your request to enter the LumApps developer program for several reasons:
 
-Needed information : 
+-   Your project cannot be done with the LumApps Marketplace.
+-   We could not verify the information you set (company website not found, company name and website do not match, etc.).
+-   The email domain does not match the company URL provided.
 
-| Information        | Description                                                                                                                                                |
-| ------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| Organization name  | The name of your organization.                                                                                                                             |
-| Organization URL   | URL of your organization’s website. The organization URL domain must match the contact email domain. We may reject your registration if it does not match. |
-| Organization email | Your support email address used to be contacted by LumApps or customers for support.                                                                        |
-| Phone Number       | A phone number to reach your support. This information is not mandatory.                                                                                   |
+## Register your Partner organization
 
-### Why my request to enter LumApps Market may be rejected?
-We may reject your request to enter the LumApps marketplace program for several reasons: 
- - The organization name is too similar to an existing organization
- - We could not verify the information you set (organization website not found, organization name and website does not match, etc.)
- - The email domain does not match the organization URL provided
- - The project is not possible with the LumApps marketplace
+Once you are enrolled in the developer program, you need to create a partner organization. A partner organization is your identity on the LumApps Marketplace. The partner organization is listed as the owner of the extensions you create.
 
-## Manage my partner organization
-Your partner information are available in the LumApps extension playground, in the back office interface.
-As for now you cannot edit your partner information by yourself. You need to contact the LumApps marketplace support if you want to edit any of your partner organization information.   
+There are two steps to register your partner organization:
+
+-   Create your private GitHub repository dedicated your marketplace extensions.
```

Comment:
With the new Developer portal this part has been removed, we do not create a GitHub repository anymore